packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -183,8 +186,11 @@ def log_decorator_wrapper(*args, **kwargs):
                     )
                 except:
                     # log exception if occurs in function
-                    logger_obj.error(
-                        "Exception: %s", sys.exc_info()[1], extra=extra_args
+                    logger_obj.exception(
+                        "Exception: %s",
+                        sys.exc_info()[1],
+                        extra=extra_args,
```

Comment:
if you change the logger to use exception, please do so also in the case above...

Reply:
This is not what I intended to do.